src/oscar/templates/oscar/customer/emails/commtype_email_changed_body.html

```diff
@@ -1,18 +1,29 @@
 {% extends "customer/emails/base.html" %}
 {% load i18n %}
 
-{% block body %}{% autoescape off %}
-<p>
-    {% blocktrans with email=new_email %}
-    You're receiving this email because your email address has been changed to <strong>{{ email }}</strong>.
-    {% endblocktrans %}
-</p>
+{% block tbody %}{% autoescape off %}
```

Comment:
I don't think `{% autoescape off %}` is right here - this is HTML and needs to be properly escaped? Looks like a mistake from before.

Reply:
No, not needed - copied from the previous version.

Reply:
It's still there?